##HerkuleX_Info Array_RAM

int8 index
MsgHerkuleX_RAM[] pHerkuleX_RAM

